### PR TITLE
Alchemy entity

### PR DESCRIPTION
**Pull request by naomiaro**
Changing AlchemyAPI usage to entities.

### CODE REVIEW

**Comment by naomiaro** [comment]
This is for ticket SDPA-218

File: server/apps/keywords/alchemy.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def get_keywords(self, text):
         params = {'apikey': app.config['KEYWORDS_KEY_API'],
                   'outputMode': 'json'}
 
-        url = app.config['KEYWORDS_BASE_URL'] + '/text/TextGetRankedKeywords' + '?' + urllib.parse.urlencode(params)
+        url = app.config['KEYWORDS_BASE_URL'] + '/text/TextGetRankedNamedEntities' + '?' + urllib.parse.urlencode(params)
```
Comment (ioanpocol) [inline server/apps/keywords/alchemy.py]: ./apps/keywords/alchemy.py:36:121: E501 line too long (121 > 120 characters)

**Comment by ioanpocol** [comment]
:+1:

File: server/features/keywords.feature [hunk 2]
```diff
@@ -11,21 +11,14 @@ Feature: Keywords
...
+                      {"relevance": "0.915151", "text": "EU"}, 
+                      {"relevance": "0.775952", "text": "Turkey"}, 
+                      {"relevance": "0.542216", "text": "Reuters"}, 
+                      {"relevance": "0.490861", "text": "ATHENS"}, 
+                      {"relevance": "0.47575", "text": "Brussels"}, 
+                      {"relevance": "0.458178", "text": "Ankara"}, 
+                      {"relevance": "0.455677", "text": "Greece"}
                    ]
          }
         """
```
Comment (ahilles107) [inline server/features/keywords.feature]: missing new line at end of the file

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**